### ISSUE TITLE
Specto barely uses premium Real Debrid

### ISSUE THREAD

**Issue by darzur** [mrknow/specto#3]
Hi,

Just installed Specto and bound it to my Real Debrid premium account. Unfortunately SPECTO finds very few sources from RD and those are mainly HQ and lower quality sources. Checked very same movies and episodes in another addon (EXODUS) and it finds sources from RD almost exclusively (at least those of highest quality ones) what.

**Comment by URMyBudE** [comment]
I have the same issue.  What was the solution?

**Comment by mrknow** [comment]
OK I will check this today.

**Comment by milnash** [comment]
Any progress on this? Great addon but barely any 1080p Real Debrid links.

Thanks for your effort and work!

**Comment by moshep15** [comment]
specto dont find any real debrids sources 720p or 1080p  even from alluc.

**Comment by mac1202** [comment]
Any news about this for pretty popular show I only got 480P source where other addon get me 1080P source. I really like specto to find HD source as it as the best library integration.

**Comment by moshep15** [comment]
scraping problem thats the only problem specto should focus to repair hope that will come soon

**Comment by noor666** [comment]
I too have the same issue and I think I know what the issue might be. When I went to configure Specto Fork, I noticed that in the list of hosts, OBOOM, UPLOADED, FILEFACTORY, RAPIDGATOR, NITROFLARE, USERCLOUD, OPENLOAD, UPLOADROCKET & UL and many others hosts that give HD links on Exodus with Real-Debrid weren't listed. Thus, I think, the reason for only getting OPENLOAD and THEVIDEO links in HQ. Cause most of the links I get on exodus are the ones mentioned above and these don't seem to be integrated into Specto as of yet

 Please add these links so proper HD sources get scraped and listed. It would be awesome to see this in the next update. Eagerly awaiting. Till then I have to use Exodus which has its own big issue. After a while the links stop opening on untill I restart Kodi, which is super annoying but its all i have till Specto starts working with Real-Debrid as these two are the best addons that use trakt properly and make it a joy to stream shows. Eagerly awaiting the fix. Thank you very much

Regards,

Noor666

**Comment by zemely** [comment]
Yeap, joining the discussion. It used be one of my two favorite addons and not anymore, unfortunately.

Seems like comments from Noor666 above is exactly what's going on, none of these sources are available.

**Comment by zemely** [comment]
Maybe there is something we don't know. Maybe that's related to the KODI switching to https? Also, Exodus use to have it's own RD settings and now it switched to the URLResolver settings on KODI, like SALTS, maybe that's the problem? But the point from noor666 about missing scrappers looks exactly what's wrong. Maybe we need more people in this thread? Come on, guys, it used to be an awesome addon.

MrKnow, let us know what the problem is at least, please?

**Comment by BankruptEd** [comment]
I have a question for any of the people in this thread. I'm not very familiar with real debrid, so I googled it and from what I'm seeing, this is something you pay for correct? Its a premium service?

**Comment by noor666** [comment]
Yes, BankruptEd. It opens up the premium links for download and streaming from the hosting sites it provides. If you aren't getting any HD or many HD links (On Exodus), with this you do. You can use it with Salts and Elysium as well but those don't have a good trakt integration like Exodus and Specto do. I love the in progress section thsee two have thus asking mrknow for the update.

You can Register for free and see how many links you get, but they won't open untill you buy the premium service. Hope that helps

**Comment by BankruptEd** [comment]
Yes that's very helpful. Thanks much noor666. ☮️

**Comment by zemely** [comment]
@BankruptEd. Yes, 100%  agree with noor666. Yesterday I've installed Elysium and had 2-4 HD streams for a new shows or movies. Then I've activated RD scrappers and had almost 20 of the 1080 HD streams, huge difference. Specto was the same and then it wasn't, very sad, especially with TVAddons down indefinatelly

**Comment by BankruptEd** [comment]
TV Add-ons is down you say?

**Comment by BankruptEd** [comment]
Damn, so it is. I hadn't realized. What affect does that have on us? Does
that mean some add-ons affiliates with them won't work?

**Comment by zemely** [comment]
Well, if you have those add-on updated to the most recent versions, they should work for as long as they can, all depends on the scrappers. There is another repo that updates Exodus and I think it's outside of TVAddons, you can google it ,there are YouTube videos about it. Elysium  is not TVAddons repo so it should also work. I use Ororo addon, that can be installed outside of TVAddons but it's paid addon(not expensive) and the only good selection is TV Show, movies sucks. TKNorris dropped SALTS but there is another repo that will update it, not sure if you can install it outside of fusion. That's pretty much all I use.
The question is: does TVAddons down for good or they just moving stuff to another page? If they are gone for good, that's not very good

**Comment by BankruptEd** [comment]
I would strongly agree with that. I wonder what's up. Seems kinda out of
nowhere. I receive their newsletter and never saw anything about
maintenance downtime. I should get by quite nicely though. I have many
different movie/TV add-ons just for such an event. Ok then, thanks very
much for your reply. Peace zemely.

**Comment by zemely** [comment]
Well, it's not out of nowhere. They have a lawsuit from Dish, TVAddons and Zem add-on, mostly for live channels, I think but everyone got scared, I guess.  I doubt TVAddons closed forever, maybe they just moving somewhere else, I don't think they USA based but regardless, might take a few weeks before they'll be back, if they will

**Comment by BankruptEd** [comment]
Well all we can do is hope for the best then. I wasn't aware of the
lawsuit. They should get a boat and go offshore lol. Thanks again my friend.

**Comment by noor666** [comment]
This is all sad news but that's why we ask mrknow in the famous aqua song manner.

♪Mrknow, know, calling Mrknow♪
♪Mrknow,  Mrknow♪
♪Get up now..... Github now♪
♪Help us now 🙏♪

Please make Specto The best Addon once Again😍😍

**Comment by noor666** [comment]
Also quick question for Zemely. 

Does your exodus stop loading RD links after watching an episode or two of something like Fargo or some other 40 minute plus episode? ?

My exodus does this most of the time resulting in me closing and reopening Kodi. Don't know if this is just me. The link tries to load and RD allows it as it comes in my downloads history.  But while loading it just stops before opening.

**Comment by zemely** [comment]
@noor666 
 I don't think so. I watched it "heavily" last Sunday, "Fargo", BTW, last season. Watched about 5 episodes(didn't like it that much, not like the last two seasons) and had a ton of RD links every time. Before these 5 episodes, I watched "Voice" in 1080 HD RD and then "AGT" in 1080 HD RD. But  I do clear cache and providers very often.  Will check this weekend again. Try to updated it from "http://repo.cazwall.com/sc" , just add this repo from the KODI File Manager first and then like any other repo, it should update Exodus to the latest version.

**Comment by noor666** [comment]
Did the update and still the same issue. I also clear the cache and providers, do you clear it every time you use kodi or every episode???. Secondly what skin are you using, as at times I think it's the Aeon Nox Silvo Mod that I use which is causing the issue. But then it does it on the default estuary skin as well. 

Also I get all the links as well that not the issue. It just fails to load the video. But only for RD links. Something like gvideo still opens. Can't pinpoint the issue. As the links aren't dead Exodus just fails to open them. Don't even know who to contact anymore about the addon and my specific RD/Exodus issue. For now I must close quit kodi and restart it and every 2 episodes or so.

Just so you know I use Kodi 17.3 on my laptop, no custom build. Using Estuary skin. As basic as it gets.

Lastly watch the other 3 episodes of Fargo, you might start liking it. Do let me know how your exodus RD experience goes. And if the problem arises for you. Also once you complete your session could you upload your log so I can see the difference, if you want to that is.

Mrknow SORRY FOR DISCUSSING EXODUS ON YOUR PAGE. WE STILL WANT SPECTO AND ARE ROOTING FOR YOU 🙌🙌🙌. 

MAKE SPECTO GREAT AGAIN.

#MakeSpectoGreatAgain

**Comment by zemely** [comment]
Oh, so you got the links, they just don't work. I see now. Well, I will test it this weekend, going to watch "The Americans" and then "Fargo"( I mean, 3rd season is OK, just not as exited as the 1st or 2nd. 1st was great and I love the movie).. I don't run mo KODI on Windows, Android, mostly on NVIDIa Shield, not sure if that makes the difference, using default Estuary skin for KODI 17.3. I normally clear everything when I use Exodus for a while, like 6-7 hours. I don't think I have to but I just do, takes a second. How long you have this issue? I use Exodus every day for two hours, one episode from the older show(1-2 720 HD RD links, if I am lucky) and one newer("Chicago Fire", tons of RD 1080 HD links, always works) and on this box I don't clear anything for a couple weeks, works fine so far. Could that be your RD account? Bit if the work for the first two episodes then it's not RD account. Where do you live? The country, I mean? 
The log, you mean KODI log? How can I do it? Ther eis an Indigo option but I am not sure Indigo works properly.
Well, since MrKnwo ignores us, we might discuss something else until he gets back. And I don't tink the RD issue with Specto goes for so long, since last year. There were some issues but they were fixed. This new issue came up fairly recently, after new KODI version. I think ,and this is strictly my humble opinion, nothing else) it might something to do with a lot of scrappers going to https. I have one Android box with KODI 16, which doesn't support https, I will clear everything on Exodus and will see how many RD HD streams I'll get. He probably busy, maybe it needs a lot of work, I  have no idea if this hard to switch to https but it must be since they came up with the whole new KODI version

**Comment by noor666** [comment]
Hey Zemely. I live in Pakistan but I don't think that's the reason for not being able to use exodus for long. This started a when I decided to sign up for RD. Before that I used exodus as is and it gave me HD links. But when the normal free links stopped giving any HD links I went for RD. And after that is started that. It's only with exodus and Salts and Elysium both still open links. But I don't like them as much. Elysium especially since it doesn't have trakt integration and I really like know what shows I have to watch.  Like the inprogress option both exodus as spector have.

Today I have a brand new issue. Exodus isn't opening anything in the My Tv shows section, which is linked with trakt. It was working yesterday. God knows what's going on at the back end. Kodi 17.3 seems to be a pain it the ass. The 16 builds never had these issues. I wilL try installing the 18 beta today and see how that fairs against this. Let me know how testing on 16 went for both exodus and specto. 

It's just annoying to close an reopen Kodi but I like watching shows and it only takes a few minutes to do so, so I am making due.

Hopefully all these brilliant addon creators survive this shit storm and make Kodi be like the old days. AMAZING!!!!!!...

Also I use Ares Wizard to upload logs then I can download it myself from the tinyurl link it gives. Looking forward to your test results. AND OF COURSE MRKNOW's FEEDBACK, UPDATE & FIX

**Comment by zemely** [comment]
Hey noor666,
OK, as far as Exodus, it's pretty much dead. As far as RD streams, it has the same thing as Specto: 1-2 barely 720 and maybe not even RD. In addition to that, Trakt stopped working and you cannot re-authorize it and you cannot view collection or watch list and none of the searches are working at all. So, to me it's dead, I cannot used it. Seems the same thing with SALTS. It's almost like they did it on purpose so nobody would use it while this lawsuit is in place. I even had one Exodus version 3.1.15 and I removed auto update but I cleared cache and providers and it's busted. The only addon that is working 100% is Elysium but it doesn't have a trakt even though when I added my TV Shows I am watching, it has all episodes I've watched marked as watched, not sure how it saw it. So, that's the status. Oh and BTW, I watched "Fargo" and you were right, it is good. I stopped on episode 3 last week and that was the last "slow" episode, good program, I was wrong.
I will test it on Kodi 16 but I don't think it would make the difference, it's not KODI, it's addons. Looks like all developers are went underground and keep quite(Exodus and SALTS are now maintained by TVAddons developers, not their original developers, lambda and TKNorris). Elysium is not a part of TVAddons so maybe it will work until everything will settle down.
I also tried Bubbles, didn't like it that much, extremely slow and 99% of the links are dead. So, unfortunately Elysium is the only working addon at this time.
Keep me posted if you will find out more. And maybe we should switch to another forum but this one OK since nobody else is talking

**Comment by noor666** [comment]
Hi Zemely,

So I have the same bad news as you but then I have some good news as well, then some really good news.

So I had the same problem with Exodus, SALTS and obviously Specto.  Elysium has no trakt so it's annoying but works wonderfully ( Finished the last season of The Leftovers on it. But there is a fix for no trakt. The addon called MetalliQ. 

THE GOOD NEWS:
MetalliQ lets you use all the above mentioned addons or the one you pick to stream the videos listed in your trakt progress or collectoins. So I just selected elysium as my default scrapper and now it just opens links like that. It works well and is good for knowing what's next. 

THE REALLY GOOD NEWS :

So I installed a custom build that has a working SALTS. The images take long to load but trakt and scraping work just fine. The build guide is here https://youtu.be/w5nU2dvnpfQ. The build has a lot of thing's you might not use so I personally just uninstalled them except a few like real-movies (uses real debrid) and a few more. Currently using that as I like have options. Now I have SALTS and Elysium+MetalliQ, both use trakt and RD links. Also Real-Movies is also good. Contain's 4k links but my net isn't fast enough for that.

Hope this helps, and heck ya Fargo. I just hope they keep making them like the American horror story seasons. It's brilliant story writing and filmography.  What's not to  love😍 . Anyway hope this helps you out. Also let me know if you find a good forum for such a discussion. I'll join you there. Hopefully the case blows over and the wonderful people get back to their awesome hobbies😍😍😍😍.

**Comment by noor666** [comment]
EXODUS JUST UPDATED. So did salts

The repo is now 3.0 and addon is 3.1.18

Trakt is working again. Wasn't getting it but then cleared cache and providers then re authorised Trakt. Cleared it again. Closed kodi and now it works. Also getting 1080 links from RD. 

Salts version 2.0.196. Repo by k3l3vra

Url resolver version 3.0.33.01

Now let's see if I get the same issue again with exodus. Where it would stop loading streams after a while. But so far so good. Still on that you tubers build, gonna keep it as it turned out to be lucky 🍀 🍀

**Comment by zemely** [comment]
Well, this is a great news!!!!! What repo it got updated from? The Exodus? I got cazwall repo but it was under maintenance on weekend.

**Comment by noor666** [comment]
It's called Paradox Repo. It came with the custom build.

**Comment by zemely** [comment]
OK, what version of Exodus do you have now?3.1.18? That's a jump from 3.1.16 that got busted. And this is a really good news, means TVAddons developers are working. Maybe MrKnow will come back with new version(w we didn't forget you, MrKnow!!!)

**Comment by zemely** [comment]
Hey man, read this:
http://koditips.com/tvaddons-offshoregit-future-update

This site has some forums for each addon, including Exodus, I am there as zemelya (made a typo here). Lots of idiots that are not reading any replies and keep asking the same questions but we can ignore those.
As I suspected, the whole lawsuit is about live TV channels, streaming will be fine, just need some time. People saying that TraktTV deleted Exodus and SALTS API ,that's why it stop working but if it's back now, it's good

**Comment by zemely** [comment]
Do you have to wait after you created an account on their site? I have an account, I logged in but it won't take my login during wizard installation. I know sometimes you have to wait like a day. I guess will try tomorrow

**Comment by noor666** [comment]
Thanks dude. Read it and gusee we just have to wait it out. But all is good for now. Also the forums were very new, made a post or two. Also left a comment to mess with the whining idiots on the link you sent 😅

**Comment by noor666** [comment]
?? What site are you talking about??

**Comment by zemely** [comment]
For the paradox

**Comment by noor666** [comment]
Ooh. Like I said. I downloaded the custom build from the YouTube video guide. It's his build. It had all the repositories pre-installed. I didn't install the paradox repo myself.

**Comment by zemely** [comment]
But don't you need an account on their site? And use a login during the installation?

**Comment by noor666** [comment]
I don't even know what site paradox has.

I added http://http://www.dimitrology.com/repo in my file manager. 

Then followed the instructions on his YouTube video. 

https://youtu.be/w5nU2dvnpfQ

Then went here
http://www.dimitrology.com/getcode
to get the code and installed the his build.

And that's it. The all I did was update the addons. If you don't see them just do search for updates.

**Comment by zemely** [comment]
OK, fresh KODI start did it, looks like, it's installing. I am trying it on KODI 16.1, will see if it updates Exodus to a new version. I've never messed with the builds before, first time. Will let you know

**Comment by noor666** [comment]
Hopefully it'll work like a charm. If not then there is always the build I installed.  It's a 17.3 fork. But you'll have to remove a lot of bloatware addons that you'll never use like sports and live TV and music. Keep the good ones. And loose the skin. I personally use Aeon Nox Silvio Mod. It has awesome Widgets and I can go to my progres episode just by clicking UP. But it takes a while to setup.

**Comment by zemely** [comment]
Another question: what kind of build you've installed from Paradox? Family? I tried it on KODI 16.1 and Exodus doesn't work, cannot stream anything, gives an error. Cannot clear providers as well but Trakt worked. They changed it from "Authorize Exodus" to "Allow KODI to use Trakt", it's nameless now. Abdn the version is 3.1.18, probably KODI 16, it's too old and a little bit "heavy" for my my old box. Got to try it on newer box, I love to mess  with it. 
Now, what KODI 127.3 fork? I know there are KODI forks but not sure they are like KODI 17.3

**Comment by noor666** [comment]
I don't have the paradox build or anything installed from that. I have the Boom shakalaka build.

Follow the video for instructions. 

http://www.dimitrology.com/2017/06/18/best-build-on-kodi-17-boom-shakalaka-%f0%9f%94%9d/

Also exodus needs 17 minimum to scrape links from the urlresolver, since they switched to https. So I guess that's why it wasn't working. Install it and let me know if it worked.

**Comment by noor666** [comment]
Hey Zemely quick question. 

When I try to scrape for some shows on exodus, I don't get 1080p links. For example American God's S01E08. But if I use SALTS or Elysium. I get a ton ( at least 40+ links. Also happened with John Wick 2. Is this just me or are you also experiencing the same issue???

Let me know how it went and the stream crashing issue seem to have resolved itself.  I hope I don't jynx it and it remains this way

**Comment by zemely** [comment]
Hey man,
OK, will check tonight. I fixed Exodus on all my boxes and disable auto updated for now. I did try several shows and movies and they have a lot of 1080 HD. Didn't reauthorized my RD, just Trakt. You might want to clear cache and providers and do a clear KODI restart but I guess you've done it already. Will let you know

**Comment by zemely** [comment]
Neither do I. A lot of HD streams but not 1080. That's weird. Episode 7 has 11 and 17 of them.
Do you have a Flixnet addon? Need some work and no Trakt, not yet but there is an IMDB setting and there is a way to use Trakt. A little bit complicated installation (not a part of TVAddons) but looks great, movies and TV Show work, RD works, it has a SuperFavorites. Check it out. These days you need every addon that works.
How is that "American Gods"? You like it?

**Comment by noor666** [comment]
It's (American Gods) very mature and has a lot of adult content.  But the premise is an interesting one, also the production quality is top notch. So I like it as I am a fan of the weird and uncommon. Also thanks for the Flixnet suggestion. Will try it out soon. And even though it doesn't have trakt I be I could simply use MetalliQ as a work around. I use it for elysium whenever exodus fails to provide 1080p sources.

**Comment by noor666** [comment]
Back to the real DISCUSSION :

Mr.know everyone is updating their add-ons to get them to work. Still awaiting your newest update.
